Makefile

```diff
@@ -7,7 +7,7 @@ export IMAGE_REPO ?= quay.io/operator-framework/rukpak
 export IMAGE_TAG ?= latest
 IMAGE?=$(IMAGE_REPO):$(IMAGE_TAG)
 KIND_CLUSTER_NAME ?= kind
-KIND := kind
+KIND_OS = kind-linux-amd64
```

Comment:
Are we using this KIND_OS variable anywhere?

Reply:
We were using it in the previous kind target - I don't think we are anymore. Good point. Will double check then remove.